stdlib/2and3/optparse.pyi

```diff
@@ -171,7 +171,7 @@ class OptionParser(OptionContainer):
     values = ...  # type: Any
     version = ...  # type: _Text
     def __init__(self, usage: Optional[_Text] = ...,
-                       option_list: List[Option] = ...,
+                       option_list: Sequence[Option] = ...,
```

Comment:
Also possibly better as `Iterable[Option]`, it is just passed onto the same method.